assets/app/styles/_components.less

```diff
@@ -47,6 +47,10 @@
     border: 1px solid #ddd;
     position: relative;
     .osc-object-highlight();
+    h2 {
+      // Avoid clipping descenders in routes on overview.
+      line-height: 1.3;
```

Comment:
Looks like the `overflow: hidden;` style in conjunction with the patternfly line height is causing the clipping, but we need `overflow` set for `text-overflow: ellipses;`